aequilibrae/paths/traffic_assignment.py

```diff
@@ -644,3 +639,85 @@ def save_skims(self, matrix_name: str, which_ones="final", format="omx") -> None
             record.procedure = "Traffic Assignment"
             record.description = out_skims.description
             record.save()
+
+    def select_link_flows(self) -> Dict[str, pd.DataFrame]:
```

Comment:
didn't we say we were going to put these all into one giant DF for ALL traffic classes?